packages/flutter/test/widgets/overscroll_stretch_indicator_test.dart

```diff
@@ -308,4 +308,58 @@ void main() {
     await gesture.up();
     await tester.pumpAndSettle();
   });
+
+  testWidgets('Stretch does not overflow bounds of container', (WidgetTester tester) async {
+    // Regression test for https://github.com/flutter/flutter/issues/90197
+    await tester.pumpWidget(Directionality(
+      textDirection: TextDirection.ltr,
+      child: ScrollConfiguration(
+        behavior: const ScrollBehavior().copyWith(overscroll: false),
+        child: Column(
+          children: <Widget>[
+            StretchingOverscrollIndicator(
+              axisDirection: AxisDirection.down,
+              child: SizedBox(
+                height: 300,
+                child: ListView.builder(
+                  itemCount: 20,
+                  itemBuilder: (BuildContext context, int index){
+                    return Padding(
+                        padding: const EdgeInsets.all(10.0),
+                        child: Text('Index $index'),
+                    );
+                  },
+                ),
+              ),
+            ),
+            Opacity(
+              opacity: 0.5,
+              child: Container(
+                color: const Color(0xD0FF0000),
+                height: 100,
+              ),
+            )
+          ],
+        )
+      )
+    ));
+
+    expect(find.text('Index 1'), findsOneWidget);
+    expect(tester.getCenter(find.text('Index 1')).dy, 51.0);
+
+    final TestGesture gesture = await tester.startGesture(tester.getCenter(find.text('Index 1')));
+    // Overscroll the start
```

Comment:
Nit: Period here.

Reply:
Ah thanks! Yes, the goldens are expected. I added an image that would otherwise be overflowing onto the following child. :)

Reply:
```suggestion
    // Overscroll the start.
```